src/Element/Element.php

```diff
@@ -82,16 +82,6 @@ protected function getDriver()
     }
 
     /**
-     * Returns selectors handler.
-     *
-     * @return SelectorsHandler
-     */
-    protected function getSelectorsHandler()
```

Comment:
Once Mink 1.7 is released, the easiest way would probably be to bump the dependency to Mink 1.7 (which autoescapes)